code/game/objects/items/pillow.dm

```diff
@@ -113,12 +119,36 @@
 		icon_state = "pillow_[variation]_t"
 		inhand_icon_state = "pillow_t"
 
+/// Puts a brick inside the pillow, increasing it's damage
+/obj/item/pillow/proc/become_bricked()
+	bricked = TRUE
+	var/datum/component/two_handed/two_handed = GetComponent(/datum/component/two_handed)
+	if(two_handed)
+		AddComponent(/datum/component/two_handed, force_unwielded = two_handed.force_unwielded + 5, force_wielded = two_handed.force_wielded + 10)
+		force += 5
+	else
+		force += 5
+	update_appearance()
+
+/// Smothers the victim while the do_after succeeds and the victim is laying down or being strangled
+/obj/item/pillow/proc/smothering(mob/living/carbon/user, mob/living/carbon/victim)
+	while(victim)
+		if(victim.body_position == FALSE && user.grab_state <= GRAB_NECK)
```

Comment:
I've just noticed a pre-existing inconsistency with the required `grab_state` in this proc (`GRAB_NECK`) and the one in `attack_secondary` (`GRAB_AGGRESSIVE`). This can be fixed by changing the operators from `>=` and `<=` to `>` and `<` or switching the required grab states between the checks. Your choice.

Reply:
huh

Reply:
thanks ghom